ARM build fails

Hey there. Great library! I'm trying to compile this on ARM and I was surprised to see that it has no problems on amd64, but fails on ARM...

```
$ git clone git@github.com:pymaxion/geographiclib-go
$ cd geographiclib-go
$ export GOARCH=arm
$ go build ./...
# github.com/pymaxion/geographiclib-go/geodesic
geodesic/test_cases.go:676:22: cannot use 42426932221845 (untyped int constant) as int value in argument to assert.InDelta (overflows)
```

--- a/geodesic/test_cases.go
+++ b/geodesic/test_cases.go
@@ -673,7 +673,7 @@ var (
 		logic: func(t *testing.T) {
 			g, _ := NewGeodesic(6378137, 1/298.257222101)
 			r := g.InverseWithCapabilities(0, 0, 60.0832522871723, 89.8492185074635, capabilities.Area)
-			assert.InDelta(t, 42426932221845, r.S12Area, 0.5)
+			assert.InDelta(t, int64(42426932221845), r.S12Area, 0.5)
 		},
 	}
 )

